src/wtc-perspective-card.js

```diff
@@ -981,6 +986,12 @@ class ClickablePerspectiveCard extends PerspectiveCard {
    * @default false
    */
   set tweening(value) {
+    if (value !== this._tweening) {
+      this._tweenBuffer = true;
```

Comment:
If I read the problem correctly, the issue is clicking on the card as soon as it closes right?
Shouldn't this buffer be turned ON and then after a second be turned OFF when setting `_tweening` to `false` then instead?